studio/src/components/app-provider.tsx

```diff
@@ -164,6 +170,17 @@ export const AppProvider = ({ children }: { children: ReactNode }) => {
         invitations: data.invitations,
       });
 
+      window.ko?.identify(data.email, {
```

Comment:
Can we make it more obvious that this only works when the script was provided? Maybe add a condition around with a few comments?

Reply:
Done. Added reset too